backend/spec/features/admin/configuration/general_settings_spec.rb

```diff
@@ -39,5 +39,31 @@
 
       expect(page).to have_content(Spree.t(:clear_cache_ok))
     end
+
+    context "with Spree mounted to a custom path" do
+      before(:each) do |example|
```

Comment:
Unused block argument - `example`. You can omit the argument if you don't care about it.